build_python_framework_pkgs.zsh

```diff
@@ -73,6 +77,9 @@ RP_ZIP="/tmp/relocatable-python.zip"
 MP_ZIP="/tmp/munki-pkg.zip"
 echo "Creating Python Framework - $TYPE"
 
+# Setup notary item
+$XCODE_NOTARY_PATH store-credentials --apple-id "macadmins@cleverdevops.com" --team-id "9GQZ7KUFR6" --password "$NOTARY_PASS" macadminpython
```

Comment:
fairly certain `"$NOTARY_PASS"` is not going to exist by default unless you add an `env` property to the github action yaml file.